Add ExtendRestAPI and SlotFills docs (#4208)
woocommerce/woocommerce-gutenberg-products-block: docs/extensibility/available-endpoints-to-extend.md

# Available endpoints to extend with ExtendRestAPI

> To see how to add your data to Store API using ExtendRestAPI, [check this document](./extend-rest-api-add-data.md).

Reviewer: I love how you've linked over to these supporting docs right at the top here. 👍

> This is a list of Available endpoints that you can extend, if you want to add a new endpoint, [check this document](./extend-rest-api-new-endpoint.md).

Reviewer 2: ```suggestion
This is a list of available endpoints that you can extend. If you want to add a new endpoint, [check this document](./extend-rest-api-new-endpoint.md).
```

## `wc/store/cart`

> The main cart endpoint is exntesible via ExtendRestAPI, the data is available on `exntesions` key in the response.

Reviewer 2: ```suggestion
The main cart endpoint is extensible via ExtendRestAPI. The data is available via the `extensions` key in the response.
```

> ### Passes Parameters:

Reviewer: Small consistency note: I think you have `Passed Parameters` in other places.

- `data_callback`: none.
- `schema_callback`: none.

### Key:

- `CartSchema::IDENTIFIER`

## `wc/store/cart/items`

> The items endpoint, which is also available on `wc/store/cart` inside the `items` key. The data would be avalaible inside each item of the `items` array.

Reviewer 2: ```suggestion
The items endpoint, which is also available on `wc/store/cart` inside the `items` key. The data would be available inside each item of the `items` array.
```
